sdk/nodejs/npm/npm.go

```diff
@@ -21,10 +22,17 @@ var _ PackageManager = &npmManager{}
 
 func newNPM() (*npmManager, error) {
 	npmPath, err := exec.LookPath("npm")
-	instance := &npmManager{
-		executable: npmPath,
+	if err != nil {
+		if errors.Is(err, exec.ErrNotFound) {
+			return nil, errors.New("Could not find `npm` executable.\n" +
+				"Install npm from https://docs.npmjs.com/downloading-and-installing-node-js-and-npm " +
```

Comment:
I wonder if we should leave out the links here?  It would be a little awkward when the bitrot, and I don't think we have a good way to prevent that.  Maybe just "Install npm and make sure it is in your PATH." is good enough?

Similar for the other package managers.

Reply:
Yeah, that's probably better!